.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+name: Build artifacts
+
+# ==== NOTE: do not rename this yml file or the run_number will be reset ====
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+env:
+  DOTNET_NOLOGO: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 6.0.x
```

Comment:
Why not 7 ?

Reply:
This PR does not bump any versions to .NET 7